postgres-persistence/src/main/resources/db/migration_postgres_data/V13.2__workflow_index_backfill_update_time.sql

```diff
@@ -0,0 +1,4 @@
+-- Optional back-fill script to populate updateTime historically.
+UPDATE workflow_index
+SET update_time = to_timestamp(json_data->>'updateTime', 'YYYY-MM-DD"T"HH24:MI:SS.MS')::timestamp AT TIME ZONE '00:00'
```

Comment:
Further testing found that the previous scripts formatting was not interpreting hours and time zones correctly.
Postgres does not interpret the timezone Z from the json->>'updateTime', it has to be set explicitly. Also, the T has to be explicitly skipped by enclosing it in double quotes.